docker-compose.uffizzi.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+
+x-uffizzi:
+  ingress:
+    service: application
+    port: 6006
+
+services:
+  application:
+    image: "${APP_IMAGE}"
+    ports:
+      - "6006:6006"
+    restart: always
+    deploy:
+      resources:
+        limits:
+          memory: 4000m
```

Comment:
I don't think the `m` notation makes sense here. Try the default `125M` and work upwards from there.  More info here https://docs.uffizzi.com/references/compose-spec/#resources-limits-memory

Also Uffizzi ignores `resources.cpu` specifications.

Reply:
- `resources.cpu` has been removed.
- Tried following memory values
  - `125M` - app preview deployment failed - https://github.com/daramayis/f-ui-kit/actions/runs/3318915889
  -  `500M` - app preview deployment failed - https://github.com/daramayis/f-ui-kit/actions/runs/3319058470/jobs/5483736212
  - `1000M` - app preview deployment failed - https://github.com/daramayis/f-ui-kit/actions/runs/3319278317
  - `2000M` - app preview deployed successfully in 8 minutes.